Automattic/jetpack
# Show the Reblog button according to the toggle in the sidebar

Fixes https://github.com/Automattic/wp-calypso/issues/85895

## Proposed changes:
The reblog button was hidden in a previous PR until we had the time to code the changes properly. Now it's time to show it again. It will show in the preview (yes, this PR only shows it in the editor view) according to the toggle value in the sidebar.

This PR removes some functions (and their files) as the endpoint should be called on publishing or updating the post, not on changing the settings in the sidebar.

### Other information:


## Jetpack product discussion
No.

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
- Apply this PR.
- This change only works for the simple sites, so we need to simulate that. To do that, go to `projects/plugins/jetpack/extensions/blocks/like/edit.js` and change line 14 from `const showReblogButton = isSimpleSite() && enabledReblogs;` to `const showReblogButton = enabledReblogs;`. Then in line 37, from `{ isSimpleSite() && (` to `{ (`.
- Enable the beta blocks by including `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );` in `projects/plugins/jetpack/jetpack.php`.
- Setup your JT site (including building the plugin).
- Create a post and include a Like block. The Reblog buttons shouldn't be there.
- Enable the Reblog button in the sidebar option and check that the reblog button is now in the Like block preview.

Thanks for the review! 🙇
